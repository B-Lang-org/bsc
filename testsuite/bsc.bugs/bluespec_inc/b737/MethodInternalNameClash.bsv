interface IFC;
   method Bit#(32) getVal();
endinterface

(* synthesize *)
module sysMethodInternalNameClash (IFC);
   method Bit#(32) getVal();
      Bit#(32) getVal = 0;
      return(getVal);
   endmethod
endmodule

