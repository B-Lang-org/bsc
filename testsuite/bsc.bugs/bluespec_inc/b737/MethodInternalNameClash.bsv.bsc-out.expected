checking package dependencies
Error: "MethodInternalNameClash.bsv", line 8, column 16: (T0011)
  Declaration of `getVal' conflicts with previous declaration at
  "MethodInternalNameClash.bsv", line 7, column 20
